tests/conftest.py

```diff
@@ -381,7 +381,7 @@ def finalizer():
         """
         Delete the resources created in the test
         """
-        for instance in instances:
+        for instance in instances[::-1]:
```

Comment:
what it will do?

Reply:
In this case, I am just using the teardown factory and the flow is as below,
teardown_factory(cbp_obj) -> teardown_factory(pvc_obj) -> teardown_factory(pod_obj)

All the elements are getting appended to a list in the above order, so the pvc is getting deleted before pod delete which is making PVC to stuck at terminating state.
So, I am reversing the list in the teardown_factory function so that pod is deleted first followed by pvc.
Please let me know if any other better solution suites here.

Reply:
Nice catch!